app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -859,7 +905,8 @@ class NewMessageViewModel @Inject constructor(
         }
     }
 
-    private fun Draft.getWholeBody(): String = uiBody.textToHtml() + (uiSignature ?: "") + (uiQuote ?: "")
+    private fun assembleWholeBody(body: String, signature: String?, quote: String?): String =
+        body + (signature ?: "") + (quote ?: "")
```

Comment:
This function can be replaced by `"$body$signature$quote"` and is only used once, so it's not really worth keeping it as a function I think